src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -1,75 +1,18 @@
-    </ProjectReference>
-    <ProjectReference Include="linux\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="debian\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="debian\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>armel</Platform>
-    </ProjectReference>
-    <ProjectReference Include="fedora\23\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="fedora\24\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="opensuse\13.2\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="opensuse\42.1\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="rhel\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="ubuntu\14.04\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="ubuntu\14.04\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>arm</Platform>
-    </ProjectReference>
-    <ProjectReference Include="ubuntu\16.04\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="ubuntu\16.04\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>arm</Platform>
-    </ProjectReference>
-    <ProjectReference Include="ubuntu\16.10\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="alpine\3.4.3\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="tizen\4.0.0\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>armel</Platform>
-    </ProjectReference>
-    <ProjectReference Include="osx\Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
-      <Platform>amd64</Platform>
+  
+  <ItemGroup Condition="'$(IsLineupPackage)' == 'true'">
+    <ProjectReference Include="..\Microsoft.NETCore.Jit\Microsoft.NETCore.Jit.pkgproj">
+      <AdditionalProperties>%(ProjectReference.AdditionalProperties);IsLineupPackage=false</AdditionalProperties>
     </ProjectReference>
   </ItemGroup>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+
+  <Import Condition="'$(_packageTargetOSGroup)' != ''"  Project="$(MSBuildThisFileDirectory)runtime.$(_packageTargetOSGroup).$(MSBuildProjectName).props" />
+  
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\dir.targets" />
```

Comment:
Did you look into using some sort of property to differentiate behavior, rather than importing different targets?  I really think this type of difference is confusing, especially with no comments.  We should have a consistent pattern followed by all projects.